sites/partners/src/components/listings/ListingFormActions.tsx

```diff
@@ -77,6 +77,10 @@ const ListingFormActions = ({
     (jurisdiction) => jurisdiction.id === listing?.jurisdictions?.id
   )
 
+  const showCloseButton = listingJurisdiction?.featureFlags?.find(
+    (flag) => flag.name === "showCloseListingButton"
+  )?.active
```

Comment:
suggestion: There was a helper function created to handle this called "doJurisdictionsHaveFeatureFlagOn". It is part of the AuthContext. If you think it makes sense we could use it here